forge-gui/res/cardsfolder/upcoming/knight_errant_of_eos.txt

```diff
@@ -0,0 +1,9 @@
+Name:Knight-Errant of Eos
+ManaCost:4 W
+Types:Creature Human Knight
+PT:4/4
+K:Convoke
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigDig | TriggerDescription$ When CARDNAME enters the battlefield, look at the top six cards of your library. You may reveal up to two creature cards with mana value X or less from among them, where X is the number of creatures that convoked CARDNAME. Put the revealed cards into your hand, then shuffle.
+SVar:TrigDig:DB$ Dig | DigNum$ 6 | ChangeNum$ 2 | Optional$ True | ChangeValid$ Creature.cmcLEX | RestRandomOrder$ True | ForceRevealToController$ True
```

Comment:
Shuffle missing

Reply:
Are you sure it will shuffle now...?

Reply:
I'm not sure, this is how similar cards do it but didn't hear the shuffle sound

Reply:
I might just add db shuffle after ?

Reply:
- If not a single Dig Effect script uses Shuffle$ param
- DigEffect class doesn't have it either

those are some pretty good hints it can't just work

Reply:
Thanks for the feedback, i'll get it fixed